packages/prettier-config/README.md

```diff
@@ -10,10 +10,10 @@ This is part of the default [open-wc](https://open-wc.org/) recommendation
 
 ## Setup
 ```bash
-npm i -g yo
-npm i -g generator-open-wc
-
-yo open-wc:linting-prettier
+npm init open-wc
```

Comment:
not sure if we wanna offer this deep of a menu... i think it's too confusing... if you want to run only this specific generator you can only do it via providing the actual generator name for it e.g. `npm init open-wc linting-prettier`

Reply:
yeah, i'll clean this up tomorrow, it was more of a draft rn :)